draft-ietf-quic-qpack.md

```diff
@@ -312,7 +312,7 @@ permit the encoder to track the decoder's state.  These events are:
 
 - Complete processing of a header block
 - Abandonment of a stream which might have remaining header blocks
-- Receipt of new dynamic table entries
+- Receipt of new dynamic table entries (Table State Synchronize)
```

Comment:
Why specify the instruction for this event, but not for the others?  This ought to be consistent.